ebs_deploy/__init__.py

```diff
@@ -426,17 +431,23 @@ def wait_for_environments(self, environment_names, health=None, status=None, ver
             environment_names = [environment_names]
         environment_names = environment_names[:]
 
-        # print some stuff
-        s = "Waiting for environment(s) " + (", ".join(environment_names)) + " to"
-        if health is not None:
-            s += " have health " + health
-        else:
-            s += " have any health"
-        if version_label is not None:
-            s += " and have version " + version_label
-        if status is not None:
-            s += " and have status " + status
-        out(s)
+        wait_time_secs = self.TIMEOUT_PER_INSTANCE * instance_count
+
+        msg_tmpl = ('Waiting for environment(s) {env_names} '
```

Comment:
Ah you're right, it's either that or backslash. I'm used to backslash probably because that's how PyCharm splits it when needed, but it's good either way. Feel free to disregard my previous comment ;)